charts/zeebe-benchmark/values.yaml

```diff
@@ -153,6 +153,18 @@ camunda-platform:
     # LogLevel defines the log level which is used by the gateway
     logLevel: debug
 
+    # Zeebe config
+    extraVolumes:
+      - name: zeebe-config
+        configMap:
+          name: zeebe-config
+          defaultMode: 0754
+
+    extraVolumeMounts:
+      - name: zeebe-config
+        mountPath: /usr/local/zeebe/config/application.yaml
```

Comment:
:thinking: Hmmm for standalone gateway we have a different configuration or?

Reply:
all our config is either under `zeebe.broker` or under `zeebe.gateway`, right? I don't see much use in offering two different values.

Reply:
Yeah sorry I guess makes sense since we have a general zeebe config section now https://github.com/zeebe-io/benchmark-helm/blob/main/charts/zeebe-benchmark/values.yaml#L27-L28 

sorry I thought first we have some separation and was not sure whether it will work together. But yes I think it is fine.